marimbabot_bringup/launch/marimbabot.launch

```diff
@@ -1,7 +1,7 @@
 <launch>
   <include file="$(find marimbabot_vision)/launch/marimbabot.launch"/>
-  <include file="$(find marimbabot_audio)/launch/marimbabot.launch"/>
-  <include file="$(find marimbabot_planning)/launch/marimbabot.launch"/>
   <include file="$(find marimbabot_speech)/launch/marimbabot.launch"/>
   <include file="$(find marimbabot_behavior)/launch/marimbabot.launch"/>
+  <include file="$(find marimbabot_audio)/launch/marimbabot.launch"/>
```

Comment:
Why did you change the order?

Reply:
they though it's much intuitive.  I forgot who made this change，since they decide to do this,  I just put it here. But from my perspective its doesnt matter.